assets/js/pages/ClusterDetails/SBDDetails.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { sbdDevicesFactory } from '@lib/test-utils/factories';
+
+import { capitalize } from 'lodash';
+
+import SBDDetails from './SBDDetails';
+
+describe('SBDDetails', () => {
+  it('should render empty state message when sbd devices are empty', () => {
+    const expectedEmptyStateMsg = 'No additional fencing details to display.';
+    const sbdDevices = [];
+
+    render(<SBDDetails sbdDevices={sbdDevices} />);
+    expect(screen.getByText(expectedEmptyStateMsg)).toBeInTheDocument();
+  });
+
+  it.each([['healthy'], ['unhealthy']])(
```

Comment:
is this list of lists intentional? `it.each(['healthy', 'unhealthy'])(...` looks more normal.
I even think the test works by accident, as you pass a list to `status` in the factory